readme.md

```diff
@@ -566,6 +566,7 @@
 - [Ledger](https://github.com/sfischer13/awesome-ledger) - Double-entry accounting on the command-line.
 - [Uncopyright](https://github.com/johnjago/awesome-uncopyright) - Public domain works.
 - [Crypto Currency Tools & Algorithms](https://github.com/kennethreitz/awesome-coins) — Digital currency where encryption is used to regulate the generation of units and verify transfers.
+- [Diversity](https://github.com/folkswhocode/awesome-diversity) - Resources about diversity in technology to create a more rich and diverse tech community.
```

Comment:
`Resources` is implied with it being an Awesome list.

I think this would be good:

> Creating a more inclusive and diverse tech community.